### DISCUSSION TITLE
Bug or user error? Adding vertical line where x-axis is Luxon/time

### DISCUSSION THREAD

**Discussion by rbubley** [chartjs/chartjs-plugin-annotation#426] [Q&A] [answered]
I have been trying to create a simple line chart with the x-axis being Luxon time (with the Luxon adaptor), with a vertical line as an annotation.

I haven't found the documentation terribly clear on what the parameters should be for the annotation, but have tried every combination of value/endValue, xMin/xMax I can think of to no avail.

JS fiddle [here:](https://jsfiddle.net/Bubz0/7xg8q623/1/)

Help would be welcome.

**Comment by kurkle** [comment, marked as answer]
You need a `scaleID` for the line to know what scale it should use. And it looks like `luxon` values do not work, you need the timestamp 
 = `.valueOf()`

https://jsfiddle.net/zq1heruv/
